meta-ostro/recipes-image/images/ostro-image.bb

```diff
@@ -182,7 +182,7 @@ COMPRESS_DEPENDS_zip = "zip-native"
 
 # Replace the default "live" (aka HDDIMG) images with whole-disk images
 # XXX Drop the VM hack after taking care also of the non UEFI devices (those using U-Boot: edison and beaglebone)
-OSTRO_VM_IMAGE_TYPES ?= "dsk dsk.xz dsk.vdi"
+OSTRO_VM_IMAGE_TYPES ?= "dsk dsk.sha256sum dsk.bmap dsk.xz dsk.vdi dsk.vdi.sha256sum dsk.vdi.bmap"
```

Comment:
In my tests they some times have about 1% of sparseness, but that's not important, or potentially can be because of issues with loosing sparseness on dsk image creation time. Anyway, setting here is just example at the moment, to validate that it works correctly in automated build. Actual settings would be adjusted later via ostroproject-ci.inc.